src/plugins/auto/src/auto_schedule.cpp

```diff
@@ -263,33 +266,35 @@ void AutoSchedule::try_to_compile_model(AutoCompileContext& context, const std::
     bool cur_dev_is_gpu = (device.find("GPU") != std::string::npos);
     {
         std::lock_guard<std::mutex> lock(m_context->m_mutex);
+        // user does not set the compiling threads
+        // limit the threads num for compiling
         if (cur_dev_is_gpu && m_compile_context[CPU].m_is_enabled) {
-            // user does not set the compiling threads
-            // limit the threads num for compiling
-            int max_threads = 0;
-            try {
-                max_threads = m_context->m_ov_core->get_property(device, ov::compilation_num_threads);
-            } catch (const ov::Exception&) {
-                LOG_DEBUG_TAG("cannot get MAX_NUM_THREADS from GPU");
-            }
-            if (max_threads == static_cast<int>(std::thread::hardware_concurrency())) {
-                int thread_num = max_threads / 2;
-                device_config.insert(ov::compilation_num_threads(thread_num));
-                LOG_DEBUG_TAG("gpu streams number for compiling: %d", thread_num);
-            } else {
-                // user set the compiling threads num
-                // use the user's val anyway
-                LOG_DEBUG_TAG("user defined compiling threads: %d", max_threads);
-            }
+            device_config.insert(ov::intel_gpu::hint::host_task_priority(ov::hint::Priority::HIGH));
```

Comment:
If user set one of `ov::intel_gpu::hint::host_task_priority` and `ov::compilation_num_threads` already, AUTO should not  modify both.

Reply:
the code logic here will be failed to set or modify `ov::intel_gpu::hint::host_task_priority` and `ov::hint::Priority::HIGH` if user has already set those properties.

Reply:
Updated.